src/main/java/frc/robot/commands/Autos.java

```diff
@@ -4,14 +4,13 @@
 
 package frc.robot.commands;
 
-import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 
 public final class Autos {
   /** Example static factory for an autonomous command. */
-  public static Command exampleAuto(ExampleSubsystem subsystem) {
-    return Commands.sequence(subsystem.exampleMethodCommand(), new ExampleCommand(subsystem));
+  public static Command exampleAuto() {
```

Comment:
quick question: is the `Commands.waitSeconds(1)` just to mock the code for what should be done in auto?

Reply:
Yes.   I wanted there to be a minimum of one auto command, and had to remove the existing example.

Reply:
ok just had to confirm as that was what I was thinking